spring-cloud-contract-verifier/src/main/java/org/springframework/cloud/contract/verifier/messaging/amqp/SpringAmqpStubMessages.java

```diff
@@ -84,10 +86,14 @@ public SpringAmqpStubMessages(RabbitTemplate rabbitTemplate, MessageListenerAdap
 
 	@Override
 	public void send(Message message, String destination) {
-		if (this.messageListenerAdapter == null) {
-			throw new IllegalStateException("no MessageListenerAdapter wired - cannot send message");
+		List<SimpleMessageListenerContainer> listenerContainers = this.messageListenerAccessor.getListenerContainersForDestination(destination);
+		if (listenerContainers.isEmpty()) {
```

Comment:
Now I wonder if we shouldn't throw an exception instead of presenting a warn. If there's no listener then sth's really wrong isn't it? I'll have to check other implementations and see if we shouldn't unify this in all of them. WDYT? For now we can leave it like this and then I'll make another issue to start throwing exceptions